styles/pypi/catppuccin.user.css

```diff
@@ -0,0 +1,1025 @@
l;stroke-miterlimit:4;stroke-dasharray:none" transform="translate(573.74 -156.864) scale(2.92733)"/></g><g transform="translate(-1683.66 -513.275) scale(.65177)"><g style="stroke:@{subtext0};stroke-width:.07269443;stroke-linejoin:bevel;stroke-miterlimit:4;stroke-dasharray:none"><path d="m-186.329 59.726 3.185 1.16v3.742l-3.185-1.159z" style="@{text}clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;@{text}solid-opacity:1;stroke:@{subtext0};stroke-width:.07269443;stroke-linecap:butt;stroke-linejoin:bevel;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" transform="translate(3477.852 524.633) scale(4.49135)"/><path d="M-189.56 60.903v3.743l3.231-1.177v-3.743z" style="@{text}clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;@{text}solid-opacity:1;stroke:@{subtext0};stroke-width:.07269443;stroke-linecap:butt;stroke-linejoin:bevel;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" transform="translate(3477.852 524.633) scale(4.49135)"/><path d="m-189.56 64.646 3.184 1.159 3.232-1.177-3.185-1.159z" style="@{text}clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;@{text}solid-opacity:1;stroke:@{subtext0};stroke-width:.07269443;stroke-linecap:butt;stroke-linejoin:bevel;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" transform="translate(3477.852 524.633) scale(4.49135)"/><path d="m-189.56 60.903 3.184 1.159 3.232-1.177-3.185-1.159z" style="fill:@{blue};fill-opacity:1;stroke:@{subtext0};stroke-width:.07269443;stroke-linejoin:bevel;stroke-miterlimit:4;stroke-dasharray:none" transform="translate(3477.852 524.633) scale(4.49135)"/><path d="M-186.376 62.062v3.743l3.232-1.177v-3.743z" style="fill:@{blue};fill-opacity:1;stroke:@{subtext0};stroke-width:.07269443;stroke-linejoin:bevel;stroke-miterlimit:4;stroke-dasharray:none" transform="translate(3477.852 524.633) scale(4.49135)"/><path d="m-189.56 60.903 3.184 1.159v3.743l-3.185-1.16z" style="fill:@{blue};fill-opacity:1;stroke:@{subtext0};stroke-width:.07269443;stroke-linejoin:bevel;stroke-miterlimit:4;stroke-dasharray:none" transform="translate(3477.852 524.633) scale(4.49135)"/></g><circle cx="2816.017" cy="1772.922" r="2.967" style="fill:@{text};fill-opacity:1" transform="skewY(-20) scale(.9397 1)"/></g></svg>';
```

Comment:
I've seeing a lot of `style="@{text}clip-rule:nonzero;` which isn't valid...

Reply:
Also, I recommend using a tool like https://www.svgviewer.dev/ to paste the SVG contents into and optimize it (shrink the size) before putting it in the userstyle.

Reply:
i took the svg from the logo itself (right click, open in new tab, then view as svg in inspect) and it came with all of this. I also removed a bunch of junk css already, there's still a bunch more. i can go through this in a few minutes to remove anything stupid

Reply:
The invalid stuff is what you put in xD. And yeah the SVGs on most websites are unoptimized for some reason so you can usually get it down to about 50% of the size or better.

Reply:
> The invalid stuff is what you put in xD.

shhh